Codecademy/gamut
# chore: update tracking package

### Overview

Below I've put the PRs of the place where PR I am using this tracking update in. If that's not okay, I can open up separate PRs applying these package updates!


Updated tracking library to v 1.0.4


### PR Checklist

- [x] Related to designs: [figma](https://www.figma.com/file/tVAwfwrr5jS83lqk3GRfEA/Trial-extension?node-id=178%3A7535&viewport=919%2C67%2C0.09)
- [x] Related to JIRA ticket: [GRW-321](https://codecademy.atlassian.net/browse/GRW-321)

#### Testing Instructions

1. Go to any of the envs below and click around (you should still see tracking events happening in the network tab) or see that lint packages is passing!



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33444 '[GRW-321] Trial Feedback Page')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/678 'Tracking library update in the mono repo')   |
| Author | [Author Link](https://github.com/codecademy-engineering/author/pull/2436) |




[GRW-321]: https://codecademy.atlassian.net/browse/GRW-321?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
